docs/en/query_language/functions/string_search_functions.md

```diff
@@ -2,12 +2,89 @@
 
 The search is case-sensitive by default in all these functions. There are separate variants for case insensitive search.
 
-## position(haystack, needle), locate(haystack, needle)
+## position(haystack, needle) {#position}
 
-Search for the substring `needle` in the string `haystack`.
-Returns the position (in bytes) of the found substring, starting from 1, or returns 0 if the substring was not found.
+Returns the position (in bytes) of the found substring `needle` in the string `haystack`, starting from 1. Returns 0 if the substring was not found.
```

Comment:
why do you repeat parameters and return values twice? (here and below)

Reply:
Removed